setup.cfg

```diff
@@ -1,2 +1,58 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+name = observatory-platform
+author = Curtin University
+author-email = agent@observatory.academy
+summary = The Observatory Platform is an environment for fetching, processing and analysing data to understand how well universities operate as Open Knowledge Institutions.
+    The Observatory Platform is built with Apache Airflow and includes DAGs (workflows) for processing: Crossref Metadata, Fundref, GRID, Microsoft Academic Graph (MAG) and Unpaywall.
```

Comment:
I guess it's just copied from setup.py, but maybe we should leave out exactly which DAGs it includes or mention some as examples, since this will change frequently.

Reply:
I guess an application of one of these things (wouldn't recommend filter branch though unless extreme care is taken):
https://stackoverflow.com/questions/2919878/git-rewrite-previous-commit-usernames-and-emails
I'm not entirely sure why those artifacts have shown up.  My guess would be that people maybe changed their local git configuration at some point.  I've disabled generation of AUTHORS file in the latest commit, so I guess it doesn't really even matter.

The email in user agent string is actually not very common, but if it is supplied, usually it is some bug reporting email for the project in case our crawler code is breaking something and the websites want to be angry at someone/report a bug. From that perspective it makes sense to supply a project email.  There is the edge case when a user downloads our code, breaks it, resulting in people still sending emails to us, but I'm not sure if that would actually be a big problem.

FYI, some example real bot user agents:
Mozilla/5.0 (Linux; Android 7.0;) AppleWebKit/537.36 (KHTML, like Gecko) Mobile Safari/537.36 (compatible; PetalBot;+https://aspiegel.com/petalbot)

- Mozilla/5.0 (compatible; Linux x86_64; Mail.RU_Bot/2.0; +http://go.mail.ru/help/robots)
- Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.1 Safari/605.1.15 (Applebot/0.1; +http://www.apple.com/go/applebot)
- Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) 
- Chrome/85.0.4183.140 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
- Mozilla/5.0 (compatible; DotBot/1.1; http://www.opensiteexplorer.org/dotbot, help@moz.com)

Documentation note: great idea. Will make the change.

Reply:
Thanks for the explanation :-)

Reply:
Made the documentation change. Need another review blessing :)